src/X.Extensions.Logging.Telegram/TelegramLogger.cs

```diff
@@ -35,14 +40,31 @@ public void Log<TState>(LogLevel logLevel, EventId eventId, TState state, Except
             throw new ArgumentNullException(nameof(formatter));
```

Comment:
This check should be moved to CTOR as if we can't resolve the formatter on startup, it makes no sense to let the app run (we can't fix this), or if we see that the formatter is empty, we can set a default one on the constructor if one not passed

Reply:
Done.